### DISCUSSION TITLE
help me write rectangular object detection code with python

### DISCUSSION THREAD

**Discussion by MARUCO26** [community/community#112403] [Programming Help]
### Body

I would like to detect the photo below using python
I tried canny or sobel, but nothins is detected I think because the variable numbers inside thoes filters are not appropriate.

I would like to detect the blue object and display it as a square centered on the blue objetct's center.
Is there anyone who can write the code?
(I want to use a classic machine vision algorithm rather than deep learning.)

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by MARUCO26** [comment]
you can check the image in this site

https://kin.naver.com/qna/detail.naver?d1id=1&dirId=10405&docId=466207971

**Comment by iAmTheBest4222** [comment]
u can use cv2 in python 
-Convert the image to the HSV color space.
-Threshold the image to isolate the blue regions.
-Find contours of the blue regions.
-Calculate the center of each contour and draw a square around it.
